scripts/migrate_project_mailing_lists.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+"""Create mailing lists for all top-level projects
+"""
+import logging
+import sys
+
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+
+from website.app import init_app
+from website.models import Node
+from website.notifications.model import NotificationSubscription
+from website.notifications.utils import to_subscription_key
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+
+EVENT = 'mailing_list_events'
+SUBSCRIPTION_TYPE = 'email_transactional'
+
+def get_targets():
+    return list(Node.find())
+
+def migrate(dry_run=True):
+    successful_enables = []
+    successful_disables = []
+    unknown_failures = {}
```

Comment:
`unknown_failures` will be handling borked data. On an ideal database, it won't be hit. With staging/prod data, it might be. The typical approach here is that if it works with relatively clean data, Steve will test it with local dumps from either staging or prod, and if any problems are discovered they're addressed then. (At least, that's been my experience with addon refactoring)